examples/example_2.mff/movieSyncs1.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Interesting - do we know if MFFpy as-is can handle movieSync files?

If not, we may need to open an issue to add another XML subclass to handle this type of file.